tree/ntuple/v7/inc/ROOT/RNTupleUtil.hxx

```diff
@@ -173,8 +173,8 @@ public:
 /// Generic information about the physical location of data. Values depend on the concrete storage type.  E.g.,
 /// for a local file `fPosition` might be a 64bit file offset. Referenced objects on storage can be compressed
 /// and therefore we need to store their actual size.
-/// TODO(jblomer): consider moving this to `RNTupleDescriptor`
-struct RNTupleLocator {
+class RNTupleLocator {
+public:
```

Comment:
Hm, what's the advantage of a `class` if we have getters and setters without further checks? Looking at the changes in this commit, a `struct` feels more ergonomic...

Reply:
It's to be more resilient to future changes. Maybe we want to rename something, or add a check (e.g., if the type used in `fPosition` fits the locator type).

Reply:
Ok. There is still a forward declaration of `struct RNTupleLocator;` in `RPageStorageFile.hxx` that is now causing warnings...